rewrite-maven/src/test/java/org/openrewrite/maven/ChangeDependencyGroupIdAndArtifactIdTest.java

```diff
@@ -36,7 +36,9 @@ void changeDependencyGroupIdAndArtifactId() {
             "jakarta.activation",
             "jakarta.activation-api",
             null,
-            null
+            null,
+            false,
+            false
```

Comment:
What has me slightly worried is that we're changing these values here from what they were (implied `false, true`), to now explicitly setting the second to `false` to keep the test passing. That to me signals that we're changing the behavior in a way that might surprise folks using the explicit constructor, as we for instance do here:
https://github.com/openrewrite/rewrite-migrate-java/blob/05fa3e0a9e228a4d5568702e36ac62ee0d4cf889/src/main/java/org/openrewrite/java/migrate/javax/AddJaxbRuntime.java#L221-L224

Can you explain why this change in default is needed, and what changes we should be making downstream as a result?

Reply:
What I'm mostly trying to look out for is that we don't inadvertently start to see more changes that we hadn't anticipated or desired following this change; and that's hard to say with just a single test added and the defaults flipped on three more. Just trying to make sure we can move forward with this with confidence.

Reply:
from what it looks like we're only sparsely using that recipe from Yaml; with no changes there expected (version was already used)
https://github.com/search?q=org%3Aopenrewrite+ChangeDependencyGroupIdAndArtifactId+language%3AYAML&type=code&l=YAML

Reply:
Looks like this is the only downstream usage of the outdated constructor:
- https://github.com/openrewrite/rewrite-migrate-java/pull/838